railties/lib/rails/generators/model_helpers.rb

```diff
@@ -19,7 +19,7 @@ module ModelHelpers # :nodoc:
       mattr_accessor :skip_warn
 
       def self.included(base) # :nodoc:
-        base.class_option :force_plural, type: :boolean, default: false, desc: "Forces the use of the given model name"
+        base.class_option :force_plural, type: :boolean, default: false, desc: "Allow pluralized model name"
```

Comment:
I think the confusion might be from the option name seeming unrelated to the description.  Is that correct?  

In that case, I see how "Allow pluralized model name" is better.  But there is a tension between "force" and "allow".  What if we describe what it prevents?  For example:

```suggestion
        base.class_option :force_plural, type: :boolean, default: false,
          desc: "Do not singularize the model name, even if it appears plural"
```

Would that make sense?

Reply:
It looks much better description. Thanks!